aws/rust-runtime/aws-credential-types/Cargo.toml

```diff
@@ -35,3 +35,6 @@ all-features = true
 targets = ["x86_64-unknown-linux-gnu"]
 rustdoc-args = ["--cfg", "docsrs"]
 # End of docs.rs metadata
+
+[package.metadata.release-tooling]
+stable = true
```

Comment:
I think we might be able to consolidate this crate into aws-types soon. Perhaps we don't want to stabilize it.

Reply:
Addressed in 3d090a2.